### PR TITLE
remove py3.3 support

### LINKED ISSUE #390: remove python 3.3 support as it reached EoL
https://devguide.python.org/#status-of-python-branches
Python 3.3 reached End of Life in 2017-09-29 and no longer is supported in any way by mainstream. No need to bother with it.

### PR DESCRIPTION
**Pull request by rooterkyberian**
resolves #390

### CODE REVIEW

File: tests/_testutil.py [hunk 1]
```diff
@@ -218,13 +218,6 @@ def create_ssl_context(self):
...
 
-    def assertLogs(self, *args, **kw):  # noqa
-        if sys.version_info >= (3, 4, 0):
-            return super().assertLogs(*args, **kw)
-        else:
-            # On Python3.3 just do no-op for now
-            return self._assert_logs_noop()
-
     @contextmanager
     def _assert_logs_noop(self):
```
Comment (webknjaz) [inline tests/_testutil.py]: Is this used somewhere else?

File: tests/test_helpers.py [hunk 2]
```diff
@@ -21,37 +21,9 @@ def _check_ssl_dir(self):
         self.assertTrue(keyfile.exists(), str(keyfile))
         return cafile, certfile, keyfile
 
-    @pytest.mark.skipif(sys.version_info >= (3, 4),
```
Comment (webknjaz) [inline tests/test_helpers.py]: How does this relate to 3.3?

File: tests/_testutil.py (hunk 1 above)
Comment (rooterkyberian) [reply tests/_testutil.py]: nope, good catch

File: tests/test_helpers.py (hunk 2 above)
Comment (rooterkyberian) [reply tests/test_helpers.py]: description was wrong, this test was 3.3 specific, as indicated by `skipif(sys.version_info >= (3, 4)`

**Comment by rooterkyberian** [comment]
Coverage dropped by one line, not sure what exactly do with it, as it is a support for old 3.4 version (pre 3.4.4) missing `ensure_future` https://docs.python.org/3/library/asyncio-task.html#asyncio.ensure_future

**Pull request merged**